frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -13,30 +13,22 @@ interface StateProps {
 }
 
 export interface TopologyDataControllerProps extends StateProps {
-  match: RMatch<{
-    name?: string;
-  }>;
```

Comment:
Can we just accept `namespace` here instead as to not couple this component to router or url.